### DISCUSSION TITLE
Setup to use onedrive with both Personal and Business accounts

### DISCUSSION THREAD

**Discussion by stevensalvini** [abraunegg/onedrive#2330] [Q&A] [answered]
Hi - I'd like to use onedrive with both a Personal and a Business account,  Is this possible?  I have it working fine with my Personal account but I don't want to trash that while trying to set up my Business account - esp. if it turns out this isn't possible!

Apologies in advance if I;m missing the obvious!

Any pointers would be much appreciated,
Steve

**Comment by abraunegg** [comment, marked as answer]
This is all clearly detailed in the documentation. Please read: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#advanced-configuration-of-the-onedrive-free-client

**Reply by stevensalvini** [reply]
Thank you for your answer and many apologies for not finding this myself in the documentation.  Now you’ve pointed it out I see it is very clearly explained.

To help others avoid wasting your time like I have, could I suggest that you explicitly mention “using with both Personal & Business accounts” in the contents page as I didn’t think if this as an advanced usage so didn’t look in that section.

Thank you for providing and supporting this excellent tool!

Steve
